spyder/config/base.py

```diff
@@ -536,7 +536,7 @@ def translate_gettext(x):
     MAC_APP_NAME = 'Spyder-Py2.app'
 
 
-def running_in_mac_app():
+def running_in_mac_app(pyexec=None):

```

Comment:
Could we add an explanation in the docstring why this is needed?

Reply:
You mean: why the keyword argument input is needed?
Most uses of this function will only need to know if `sys.executable` (Spyder's) is inside a standalone Mac application. However, the interpreter status in the statusbar needs to know if `self._interpreter` is running in a Mac app ("Same as Spyder"), i.e. 'internal', which is not conda!

Reply:
Ok let;'s addd that nicely to the docstring :-) 

I meant the pyexec. Your explanation is perfect, just need to add it there for context :-)